mri/tests/test_reconstructors.py

```diff
@@ -245,10 +246,19 @@ def test_sparse_calibrationless_reconstruction(self):
                     n_jobs=2,
                     gradient_formulation=formulation,
                 )
+            regularizer_op_gl = GroupLASSO(weights=0)
+            linear_op.op(image_multichannel)
+            regularizer_op_owl = OWL(
```

Comment:
where is this one used?

Reply:
Oh damn I have pushed it. I am sorry for this confusion, I am still in progress , this I pushed yesterday before going home to save my changes

Reply:
@chaithyagr please think about using sparseGroupLASSO too

Reply:
Yes, @philouc I will get a PR to Modopt for this and add the test here later.

I could have done this while getting GroupLASSO in Modopt, however, I realized that it is just GroupLASSO(SparseThreshold), so I just had a chat with @sfarrens about this, I plan to implement a generalized conjugation operator in Modopt, that way we can implement SparseGroupLASSO as:

`ConjugationOperator(GroupLASSO, SparseThreshold)` 

Also, this will ensure that we can do more general conjugation operations in the future.

Reply:
@chaithyagr the composition operation is actually the right way to move on  once we're given the prox operators of GroupLASSO and SoftThresholding. I agree with you on the fact that this feature could be used too for the the implementation of the elasticNet regularization (Rdige+ L1 norm)
Let's wait for the PR accepted in ModOpt by @sfarrens .